### ISSUE TITLE
deleteApi has a problem with OffsetDateTime.MIN , OffsetDateTime.MAX

### ISSUE THREAD

**Issue by patrick-schweizer** [influxdata/influxdb-client-java#128]
Given:

```
DeletePredicateRequest predicate = new DeletePredicateRequest()
    .start( OffsetDateTime.MIN)
    .stop(OffsetDateTime.MAX)
    .predicate("...");
deleteApi.delete(predicate, configBucket, configOrg);
```
will result in 
```
com.influxdb.exceptions.BadRequestException: invalid request; error parsing request json: invalid RFC3339Nano for field stop, please format your time with RFC3339Nano format, example: 2009-01-01T23:00:00Z

	at com.influxdb.internal.AbstractRestClient.responseToError(AbstractRestClient.java:96)
	at com.influxdb.internal.AbstractRestClient.execute(AbstractRestClient.java:81)
	at com.influxdb.client.internal.DeleteApiImpl.delete(DeleteApiImpl.java:108)
```
whereas
```
        DeletePredicateRequest predicate = new DeletePredicateRequest()
                .start(OffsetDateTime.now().minusMinutes(30))
                .stop(OffsetDateTime.now())
                .predicate("...");
        deleteApi.delete(predicate, configBucket, configOrg);
```
works.

Expected behavior: 
Either the request can be executed or the error message should indicate the specific problem (I assume the range might be out of bounds).

**Comment by bednar** [comment]
@patrick-schweizer thanks for an issue, we will take a look

**Comment by bednar** [comment]
The InfluxDB strictly requires a time in range defined by [Rfc3339](https://tools.ietf.org/html/rfc3339): 

> All dates and times are assumed to be in the "current era", somewhere between 0000AD and 9999AD.

We will add a validation to our adapter that time is in required range:

https://github.com/influxdata/influxdb-client-java/blob/235f288485b8ba05919af634f8c91e671d9a2945/client/src/generated/java/com/influxdb/client/JSON.java#L181

Thanks again

**Comment by patrick-schweizer** [comment]
> We will add a validation to our adapter that time is in required range

Perfect, thank you!

**Comment by bednar** [comment]
Hi @patrick-schweizer,

The issue is fixed in [1.11.0 milestone](https://github.com/influxdata/influxdb-client-java/milestone/14). 

If you would like to use a preview version use our snapshot build from repository: https://oss.sonatype.org/content/repositories/snapshots/

Regards
